src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/ContainerBuilderTest.php

```diff
@@ -13,6 +13,7 @@
 
 require_once __DIR__.'/Fixtures/includes/foo.php';
 
+use PhpUnit\Framework\TestCase;
```

Comment:
wrong namespace. It is ``PHPUnit`` (same in many files, and it breaks on the CI server)